org.civicrm.bootstrapcivihr/scss/components/_spinner.scss

```diff
@@ -0,0 +1,3 @@
+.spinner {
+  @include spinner(1.5s);
```

Comment:
now why did you change it to 1.5s, when before it was 1s?

if *both* implementations of the spinner requires 1.5s, then just define that as the default value and don't pass anything to the function

Reply:
> now why did you change it to 1.5s, when before it was 1s?
Previously, the speed was `1s` but i realized the speed should  actually be `1.5s` as per @jon's comment. so i changed it to later.